app/controllers/personal_access_tokens_controller.rb

```diff
@@ -0,0 +1,17 @@
+class PersonalAccessTokensController < ApplicationController
```

Comment:
any reason why not to use the API?

Reply:
No, this should be done in react as well.